types/module/server/manager.go

```diff
@@ -199,6 +206,28 @@ func (mm *Manager) InitGenesis(ctx sdk.Context, genesisData map[string]json.RawM
 	return res
 }
 
+// RunMigrations performs state migrations for registered modules.
+func (mm *Manager) RunMigrations(ctx sdk.Context, cdc codec.Codec) error {
+	keys := make([]string, 0, len(mm.migrationHandlers))
+	for k := range mm.migrationHandlers {
+		keys = append(keys, k)
+	}
+	sort.Strings(keys)
```

Comment:
Might be nice to add a comment here as to why we are sorting (i.e. to prevent non-determinism).